pkg/client/gen_test.go

```diff
@@ -460,7 +463,7 @@ func TestGenerateManifestGolden(t *testing.T) {
 					},
 				},
 			},
-			goldenFile: filepath.Join("testdata", "imagePullPolicy-all-plugins.golden"),
+			goldenFile: filepath.Join("testdata", "imagePullPolicy-not-all-plugins.golden"),
```

Comment:
This was actually a previously duplicated test oddly. Now one is for forcing the values, and the other isnt.